test/powershell/Language/Scripting/ActionPreference.Tests.ps1

```diff
@@ -60,42 +61,7 @@ Describe "Tests for (error, warning, etc) action preference" -Tags "CI" {
             $error.Count | Should -BeExactly $errorCount
         }
 
```

Comment:
is there a way to add a break into the debugger test?

Reply:
Not as an automated test that I'm aware of. I've been looking at that code quite a bit and trying to determine the best way to set up debugger testing, because it would be incredibly useful for all of us. Maybe another event handler on the debugger that can be used to execute debugger commands once we're actually in the debugger. I've tried a bunch of things with what's already there, but so far I haven't had any luck. @PaulHigin mentioned an internal tool/class for testing that was not open source.

Reply:
After sleeping on the problem, I came up with a solution that allows me to programmatically test the debugger without requiring any code changes or refactoring. 😄 

Now to write up some Pester tests to automate the testing of this functionality.

Reply:
Just a quick update. I submitted a separate PR for debugger command test automation, and once that is approved/merged I'll use the module that PR includes to do automated testing in this PR.